Rmd/tgx-ddi-references.bib

```diff
@@ -0,0 +1,43 @@
+@ARTICLE{li2019review,
```

Comment:
At some point also we talked about putting bib files in a subdirectory; again not urgent but maybe some day